src/diffusers/models/autoencoders/autoencoder_kl_temporal_decoder.py

```diff
@@ -205,6 +205,7 @@ def __init__(
         sample_size: int = 32,
         scaling_factor: float = 0.18215,
         force_upcast: float = True,
+        use_legacy: bool = True,
```

Comment:
@DN6 @sayakpaul The newer SVD checkpoints like MotionCtrl and DragNUWA do not make use of the quant_conv layer. Is it okay to add a flag, maybe with a better name, in a non backwards breaking change? In the Stability AI generative-models codebase, it has been deprecated too IIRC.

Reply:
I believe the image conditioning requires using this layer right? 

https://github.com/Stability-AI/generative-models/blob/1e30a2df801a18c82a24223adfbdc71be0e8ca20/scripts/sampling/configs/svd_xt_1_1.yaml#L72C46-L72C67

Which points to:
https://github.com/Stability-AI/generative-models/blob/1e30a2df801a18c82a24223adfbdc71be0e8ca20/sgm/models/autoencoder.py#L602

Reply:
That is correct @DN6. SVD XT 1.1 still uses the AutoencodingEngineLegacy but MotionCtrl/DragNUWA do not AFAICT.

Reply:
I think we can change the flag to `use_quant_conv` instead of legacy.